docs/Makefile

```diff
@@ -1,20 +1,20 @@
-# Minimal makefile for Sphinx documentation
-#
-
-# You can set these variables from the command line.
-SPHINXOPTS    =
-SPHINXBUILD   = sphinx-build
-SPHINXPROJ    = XMLSERVICE
-SOURCEDIR     = .
-BUILDDIR      = _build
-
-# Put it first so that "make" without argument is like "make help".
-help:
-	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
-
-.PHONY: help Makefile
-
-# Catch-all target: route all unknown targets to Sphinx using the new
-# "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
-%: Makefile
+# Minimal makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line.
+SPHINXOPTS    =
+SPHINXBUILD   = sphinx-build
+SPHINXPROJ    = XMLSERVICE
+SOURCEDIR     = ./srcdir

```

Comment:
It looks like you re-ran sphinx-quickstart or something (line endings changed to CRLF) and changed the source and build directories. These shouldn't be changed and in fact it doesn't build:

```
$ make html
usage: sphinx-build [OPTIONS] SOURCEDIR OUTPUTDIR [FILENAMES...]
sphinx-build: error: config directory doesn't contain a conf.py file (/home/kadler/projects/xmlservice/docs/srcdir)
make: *** [Makefile:20: html] Error 2
```

Please revert the changes to this file and move the rst files in to the docs/ directory.

Reply:
I forgot to move conf.py into srcdir. Sorry. Now it works.

Reply:
But why is there a srcdir at all? Sphinx was already set up correctly. Please undo the change and move the rst files in to the docs directory.

Reply:
I moved *.rst back and also changed Makefile as you asked. I have new commit submitted.